data-client/src/main/java/org/wikipedia/dataclient/mwapi/MwQueryPage.java

```diff
@@ -141,6 +151,31 @@ public void appendTitleFragment(@Nullable String fragment) {
         title += "#" + fragment;
     }
 
+    public boolean checkFileInUse() {
```

Comment:
Could you please merge these two methods into a single one, and give it a more explicit name, such as `checkWhetherFileIsUsedInWikis`? Thanks!